micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmMemory.java

```diff
@@ -35,11 +35,13 @@ static Stream<MemoryPoolMXBean> getLongLivedHeapPools() {
     }
 
     static boolean isConcurrentPhase(String cause, String name) {
-        return "No GC".equals(cause) || "Shenandoah Cycles".equals(name) || "ZGC Cycles".equals(name);
+        return "No GC".equals(cause) || "Shenandoah Cycles".equals(name) || "ZGC Cycles".equals(name)
+                || (name.startsWith("GPGC") && !name.endsWith("Pauses"));
```

Comment:
Ooops, shouldn't this be `"Cycles"` instead of `"Pauses"`?
This is used to record the time spent in concurrent phase and time spent in GC pause (`jvm.gc.concurrent.phase.time`, `jvm.gc.pause`). Based on the [release notes](https://docs.azul.com/prime/release-notes#prime_stream_22_12_0_0), I think this should be `"Cycles"`:
>GPGC New/Old Cycles: duration time in ms of the concurrent GC which runs in parallel to application threads and is not stopping the application.
GPGC New/Old Pauses: GC pause time in ms.

Reply:
The condition now is starts with `GPGC` and does NOT end with `Pauses`, which would include the previous naming ("GPGC New/Old") as well as the new "GPGC New/Old Cycles". The problem with the previous naming is that it includes both concurrent and pause time. We're in a hard spot with how to count the previous naming. Either we count it as concurrent time or as pause time, but then whichever we count it as will be larger than it actually is (because it includes the other).

Reply:
>does NOT end with

Arghhh, of course I did not see the negation. :D
One more thing about the old naming: even if the latest 11 and 17 builds are fixed by default, 8 is still using the old naming.